airflow/config_templates/default_webserver_config.py

```diff
@@ -34,7 +34,7 @@
 SQLALCHEMY_DATABASE_URI = conf.get('core', 'SQL_ALCHEMY_CONN')
 
 # Flask-WTF flag for CSRF
-CSRF_ENABLED = True
+WTF_CSRF_ENABLED = True
```

Comment:
https://github.com/lepture/flask-wtf/blob/v0.14.2/flask_wtf/csrf.py#L176-L189 is what reads this. I have confirmed that setting `CSRF_ENABLE=False` does nothing, but setting `WTF_CSRF_ENABLED = False` does correctly disable the requirement for CSRF.

Reply:
:+1: :+1: :+1: